src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import Text from "src/components/Text";
+import Image from "next/image";
+
+import * as S from "./styles";
+
+const ContactUsContent: React.FC = () => {
+  const [name, setName] = useState<string | null>(null);
+  const [subject, setSubject] = useState<string | null>(null);
+  const [content, setContent] = useState<string | null>(null);
+
+  const handleSubmit = (event: React.MouseEvent) => {
+    event.preventDefault();
+
+    if (name === null) {
+      setName("");
+    }
+
+    if (subject === null) {
+      setSubject("");
+    }
+
+    if (content === null) {
+      setContent("");
+    }
+
+    if (!name || !subject || !content) {
```

Comment:
What does this line do

Reply:
If either the name, subject, or content is empty, do not send email.

Reply:
yup, actually empty in a sense that `null`, `undefined` or `""`